man/man5/zpool-features.5

```diff
@@ -14,6 +11,10 @@
 .\" CDDL HEADER, with the fields enclosed by brackets "[]" replaced with your
 .\" own identifying information:
 .\" Portions Copyright [yyyy] [name of copyright owner]
+.\" Copyright (c) 2012, 2018 by Delphix. All rights reserved.
```

Comment:
"by"?

Reply:
DAmnit, keep missing things.
Yeah one of my anoyances... Afaik Delphix doesn't make womanswear.

Reply:
proudly presented :P